Gulpfile.ts

```diff
@@ -748,7 +745,7 @@ gulp.task("runtests",
 const nodeServerOutFile = "tests/webTestServer.js";
 const nodeServerInFile = "tests/webTestServer.ts";
 gulp.task(nodeServerOutFile, false, [servicesFile], () => {
-    const settings: tsc.Settings = getCompilerSettings({}, /*useBuiltCompiler*/ true);
+    const settings: tsc.Settings = getCompilerSettings({module: "commonjs"}, /*useBuiltCompiler*/ true);

```

Comment:
I'm going to merge this without fixing these nits and the fix them when I merge the lint rule catching them, is that okay?